MVVM Module/Source/Feed Presentation/FeedViewModel.swift

```diff
@@ -20,14 +20,20 @@ final class FeedViewModel {
 
 	var onLoadingStateChange: Observer<Bool>?
 	var onFeedLoad: Observer<[FeedImage]>?
+	var onLoadError: Observer<String?>?
```

Comment:
`onLoadError` could be named `onErrorStateChange` which make the intent more clear and follows the `onLoadingStateChange` property.

```suggestion
	var onErrorStateChange: Observer<String?>?
```